src/components/home/modals/AddToken.vue

```diff
@@ -101,7 +100,9 @@ export default {
             this.$http.get(url).then(response => {
                 this.responseErr = false
                 Vue.set(this.tokens, this.tokenId, JSON.parse(JSON.stringify(this.tokenId)))
```

Comment:
get `tokenId` from http response, not `this.tokenId`. This value could be clear to empty.